torch/csrc/jit/codegen/cuda/ir_builder.cpp

```diff
@@ -321,11 +321,31 @@ Val* SimplifyingIrBuilder::divExpr(Val* lhs, Val* rhs) {
   return IrBuilder::divExpr(lhs, rhs);
 }
 
-Val* SimplifyingIrBuilder::ceilDivExpr(Val* lhs, Val* rhs) {
+Val* SimplifyingIrBuilder::ceilDivExpr(Int* lhs, Int* rhs) {
```

Comment:
Looks like this is not necessary to achieve what I want, but since I already do the work here, I will keep it.